zcash_test_vectors/orchard/commitments.py

```diff
@@ -86,5 +93,27 @@ def randbytes(l):
 
     assert value_commit(rcv, v) == VALUE_COMMITMENT_RANDOMNESS_BASE * rcv + VALUE_COMMITMENT_VALUE_BASE * v
 
+# Test consistency of ValueCommit^{Orchard} with precomputed generators for non-native asset
+def test_value_commit_zsa():
```

Comment:
remove `test_value_commit_zsa()` and simply add another assert into `value_commit_zsa()`.

Reply:
done